src/stories/Library/Modals/Modal.tsx

```diff
@@ -19,15 +19,20 @@ const Modal = ({ children, shownModal, classNames }: ModalWrapperProps) => {
     return <ModalFallbackButton toggleModal={toggleModal} />;
 
   return (
-    <div className={clsx("modal", shouldShowModal && "modal-show", classNames)}>
-      <div className="modal__screen-reader-description" id="describemodal">
-        Denne modal dækker sidens indhold, og er en demo
+    <div className="modal-backdrop">
```

Comment:
For future reference I would have preferred a different structure for the class name.

`modal-backdrop` appears as a separate block but to me it is part of the `modal` block. In that case we should have made `modal` the root element with the backdrop styling. `modal__content` would then be the element which wrapped the actual UIs.

Reply:
I agree, but I also believe that is a bridge that was crossed a long time ago by ITK, and we approved it. On top of that rubbing me the wrong way, changing it now would mean having to change the react components too, and like the previous comment, I think that deserves its own separate task.